docs/openj9_support.md

```diff
@@ -161,6 +163,48 @@ platform support might vary, depending on their build environment. Check the Ado
 OpenJDK12 binaries are supported on the minimum operating system levels shown in the following tables:
 
 
+| Linux                                 |  x64   |  ppc64le   | Z64  |
+|---------------------------------------|--------|------------|------|
+| Centos 6.9                            |   Y    |     Y      |  N   |
+| Centos 7.4                            |   Y    |     Y      |  N   |
+| Red Hat Enterprise Linux (RHEL) 6.9   |   Y    |     Y      |  Y   |
+| RHEL 7.4                              |   Y    |     Y      |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |     Y      |  Y   |
+| Ubuntu 16.04                          |   Y    |     Y      |  Y   |
+| Ubuntu 18.04                          |   Y    |     Y      |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x64   |
+|----------------------------|--------|
+| Windows 7 SP1              |   Y    |
+| Windows 8                  |   Y    |
+| Windows 8.1                |   Y    |
+| Windows 10                 |   Y    |
+| Windows Server 2012        |   Y    |
+| Windows Server 2012 R2     |   Y    |
+| Windows Server 2016        |   Y    |
+
+| macOS                   |   x64    |
+|-------------------------|----------|
+| OS X 10.9.0+            |    Y     |
+
+| AIX          |  ppc64   |
+|--------------|----------|
+| AIX 7.1 TL4  |    Y     |
+| AIX 7.2      |    Y     |
+
+When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
+
+### OpenJDK 13
```

Comment:
You could just change 12 -> 13 instead of adding a new jdk13 section, since 12 is obsolete now.

Reply:
Last year we removed 10 1Q after 11 GA'd. We'll follow the same pattern this time, removing 12 1Q after 13 GAs. There will be some tidy up work to do across the docs to ensure all the icons are updated, so we'll tackle it all together in one hit.